index.html

```diff
@@ -10,17 +10,48 @@
 </head>
 
 <body>
```

Comment:
- [ Required ] It would be better if the `margins` on the `header` and `footer` on your webpage are consistent with the wireframe provided for this project. Currently, it appears the two sections are centered yet they should have a full width across the webpage.

## Wireframe
![Screenshot from 2023-01-19 11-03-02](https://user-images.githubusercontent.com/16392046/213387426-6e3df1fb-d7f0-4dd0-8b62-5af709f4cd98.png)

## Your design
![Screenshot from 2023-01-19 10-54-42](https://user-images.githubusercontent.com/16392046/213386679-ae646aac-3d1d-4631-8b72-07a2b139d6d3.png)

Reply:
Fixed.